src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -155,7 +155,7 @@ private void RaiseAnalyzersChangedIfBindingChanged()
         {
             var newBindingConfiguration = this.configurationProvider.GetConfiguration();
 
-            if (CurrentConfiguration != newBindingConfiguration)
+            if (!CurrentConfiguration.Equals(newBindingConfiguration))
```

Comment:
The solution tracker was raising a "config changed" event when in fact it had not because it was using object identity to determine if the configuration had changed.
However, the config provider is currently reloading the config each time so the objects will always be different.